.github/scripts/auto_release.py

```
@@ -0,0 +1,364 @@
+    update_changelog(changelog_entry)
+
+    # Step 6: Build extension with new version
+    # This ensures the built artifacts contain the correct version
+    build_extension()
+
+    # Step 7: Publish to marketplaces
+    # CRITICAL: Publish BEFORE committing
+    # If publish fails, we haven't committed a broken version
+    try:
+        publish_extension()
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)